### DISCUSSION TITLE
Laser GRBL and Ortur Laser Master 2 Pro

### DISCUSSION THREAD

**Discussion by tsandcs** [arkypita/LaserGRBL#1741] [Q&A]
Hello all. the software does not seem to connect with the laser. I press the connect button, it says connecting but 10 seconds later I get the disconnect sound and it says disconnected. Please give me a way out as this is a newly purchased machine and the downloaded software is V4.7.1. There is only one COM Port to select from. The blue USB light on the laser cutter panel flashes blue for a bit and then there is no light and eventually the Ortur laser cutter switches itself off. Please help!
![image](https://user-images.githubusercontent.com/99172657/152771066-6fa75197-6200-463d-8cc8-a312d0e48ee0.png)

**Comment by arkypita** [comment]
Did you install Ortur serial driver?

**Reply by tsandcs** [reply]
Downloaded STMicro v1.564bit as I am using Windows 8

**Reply by tsandcs** [reply]
Tried to manually update driver for COM3 got this screen
![image](https://user-images.githubusercontent.com/99172657/152790592-b6c4670d-2fad-4b33-8030-79eee6d2bef9.png)

**Reply by arkypita** [reply]
What do you see in device manager under COM/LPT

**Reply by arkypita** [reply]
> Tried to manually update driver for COM3

COM3 is the phisical port of your PC, not the virtual USB serial port added by the machine. Another entry should appare under com/lpt for your engraver (called ST microelectronics...)

**Comment by tsandcs** [comment]
Have loaded under add legacy hardware in device manager, got this screen going to check now if it picks up as it has a warning sign on it
![image](https://user-images.githubusercontent.com/99172657/152791942-5039d85a-b3dc-441f-a08c-d79fabb6e960.png)

**Reply by tsandcs** [reply]
Still no joy, it does not add the new ports in the lasergrbl or lightburn although they show in device manager. Please need to resolve this issue.

**Comment by StuartB4** [comment]
You can try uninstalling the Com Port and let Windows reinstall it.

https://www.youtube.com/watch?v=cccKdNG8oYQ

**Reply by tsandcs** [reply]
Dont worry, for an online help system, you guys suck. All you had to do was tell me the chipset was not the one used in all your tutorial videos etc. I had to research that so I could get the correct com port driver, been producing since monday afternoon.

**Reply by arkypita** [reply]
So you, supreme example of intelligence, please enlighten us by writing which driver was needed.

Don't worry, as people who give their time for free in helping others (remember **this is not** ortur customer service) we can afford not to know everything.

Apart from the fact that my first answer was "did you install Ortur serial driver?" and it should have been enough already, I guess you didn't even bother to read the instructions that will have been with the engraver, where it probably said the same thing.

**Comment by tsandcs** [comment]
From where we had to download the software was only SMT drivers, which we
did download and install and nothing would work. FYI all instructions were
read on all documentation etc that came with the engraver, suggesting that
I am stupid or not a tech genius as you might think you are is uncalled
for, all I did was state a fact, whether your after sales service or help
line is free or not is irrelevant.

For the record I downloaded the Zadig 2.5 driver form Aufero and used that,
hey presto everything was hunky dory.

Kind regards

**Reply by gmmanonymus111** [reply]
I think you misunderstood. We are volunteers helping out others, and LaserGRBL is not affiliated with any laser engraver company. It is a standalone program manufacturers bundle with their machines, because, you know, it's free. I might date to say some of them won't even bother checking if it's compatible with their machines.

**Comment by Sapheus** [comment]
still doesn't work.  for my sculpfun I ad to install  driver for the Chinese chipset for Arduino ( ch340) after that no problem with lightburg. I suppose most of these laser use Arduino clones.
